NOTES-NONSTOP.md

```diff
@@ -78,12 +78,64 @@ The current OpenSSL default memory model uses the default platform address
 model. If you need a different address model, you must specify the appropriate
 c99 options for compile (`CFLAGS`) and linkers (`LDFLAGS`).
 
+Cross Compiling on Windows
+--------------------------
+
+To configure and compile OpenSSL, you will need to set up a Cygwin environment.
+The Cygwin tools should include bash, make, and any other normal tools required
+for building programs.
+
+The `PATH` must include the bin directory for the c99 cross-compiler, as in:
+
+    export PATH=/cygdrive/c/Program\ Files\ \(x86\)/HPE\ NonStop/L16.05/usr/bin:$PATH
+
+This should be set before Configure is run. For the c99 cross-compiler to work
+correctly, you also need the COMP_ROOT set, as in:
+
+    export COMP_ROOT="C:\Program Files (x86)\HPE NonStop\L16.05"
+
+`COMP_ROOT` needs to be in Windows form.
+
+`Configure` must specify the `no-makedepend` option otherwise errors will
+result when running the build because the c99 cross-compiler does not support
+the `gcc -MT` option. An example configure command is run from the OpenSSL
+directory as follows:
+
+    ./Configure nonstop-nsx_64 no-makedepend --with-rand-seed=rdcpu
+
+Do not forget to include any OpenSSL cross-compiling prefix and certificate
+options when creating your libraries.
+
+The OpenSSL test suite will not run on your workstation. In order to verify the
+build, you will need to perform the build and test steps in OSS in your NonStop
+server. You can also build under gcc and run the test suite for Windows but that
+is not equivalent.
+
+**Note:** In the event that you are attempting a FIPS-compliant cross-compile,
+be aware that signatures may not match between builds done under OSS and under
+cross-compiles as the compilers do not necessarily generate identical objects.
+Anything and everything to do with FIPS is outside the scope of this document.
```

Comment:
It would be nice to include a reference to the security policy but that's not available yet.  What about adding: _Refer to the FIPS security policy._ or is that going to cause too much pain if it doesn't contain a link?

Reply:
I'll make a note to myself to update the link once the policy is created, or we can do that during the PR for the FIPS security policy itself.